codegen/templates/tchannel_client.tmpl

```diff
@@ -166,51 +166,31 @@ func {{$exportName}}(deps *module.Dependencies) Client {
 			configureCircuitBreaker(deps, timeoutVal, circuitBreakerName, qpsLevel)
 		}
 	}
-
-	var client *zanzibar.TChannelClient
-
+	var maxAttempts int
 	if  deps.Default.Config.ContainsKey("tchannelclients.retryCount.feature.enabled") && deps.Default.Config.MustGetBoolean("tchannelclients.retryCount.feature.enabled") && deps.Default.Config.ContainsKey("clients.{{$clientID}}.retryCount") && int(deps.Default.Config.MustGetInt("clients.{{$clientID}}.retryCount")) > 0{
-		maxAttempts := int(deps.Default.Config.MustGetInt("clients.{{$clientID}}.retryCount"))
-		client = zanzibar.NewTChannelClientContext(
-				channel,
-				deps.Default.ContextLogger,
-				deps.Default.ContextMetrics,
-				deps.Default.ContextExtractor,
-				&zanzibar.TChannelClientOption{
-					ServiceName:          serviceName,
-					ClientID:             "{{$clientID}}",
-					MethodNames:          methodNames,
-					Timeout:              timeout,
-					TimeoutPerAttempt:    timeoutPerAttempt,
-					RoutingKey:           &routingKey,
-					RuleEngine:           re,
-					HeaderPatterns:       headerPatterns,
-					RequestUUIDHeaderKey: requestUUIDHeaderKey,
-					AltChannelMap:        altChannelMap,
-					MaxAttempts:          maxAttempts,
-				},
-			)
+		maxAttempts = int(deps.Default.Config.MustGetInt("clients.{{$clientID}}.retryCount"))
 	}else{
-		client = zanzibar.NewTChannelClientContext(
-				channel,
-				deps.Default.ContextLogger,
-				deps.Default.ContextMetrics,
-				deps.Default.ContextExtractor,
-				&zanzibar.TChannelClientOption{
-					ServiceName:          serviceName,
-					ClientID:             "{{$clientID}}",
-					MethodNames:          methodNames,
-					Timeout:              timeout,
-					TimeoutPerAttempt:    timeoutPerAttempt,
-					RoutingKey:           &routingKey,
-					RuleEngine:           re,
-					HeaderPatterns:       headerPatterns,
-					RequestUUIDHeaderKey: requestUUIDHeaderKey,
-					AltChannelMap:        altChannelMap,
-				},
-			)
+		maxAttempts = 5
```

Comment:
Declare a const int with the right name. Magic numbers will not pass lint

Reply:
+1